setup.py

```diff
@@ -1,22 +1,17 @@
 from setuptools import setup, find_packages
 
+def get_dependencies():
+    """Reads the dependencies from the requirements file."""
+    with open('requirements.txt', 'r') as d:
+        dependencies = d.read()
+
+    return dependencies
+
 setup(
     name='hotsos',
     version='1.0.0',
     packages=find_packages(include=['hotsos*']),
-    install_requires=[
-        'click',
-        'pyyaml',
-        'progress',
-        'simplejson',
-        'structr',
-        'fasteners',
-        # Pin to cryptography==3.4.8 since the cryptography-3.4.x branch is the
-        # last branch that can be built without the Rust toolchain.
-        # More info on issue: https://github.com/canonical/hotsos/issues/326
-        # [TODO] Bump cryptography to more recent release
-        'cryptography==3.4.8',
-    ],
+    install_requires=get_dependencies(),
     entry_points={
       'console_scripts': [
         'hotsos=hotsos.cli:main']
```

Comment:
might as well cleanup the snapcraft.yaml while we're at it:

```
diff --git a/snap/snapcraft.yaml b/snap/snapcraft.yaml
index 280fc942..07f2074d 100644
--- a/snap/snapcraft.yaml
+++ b/snap/snapcraft.yaml
@@ -38,10 +38,8 @@ parts:
   hotsos:
     plugin: python
     source: .
-    stage-packages: [coreutils, bsdmainutils, systemd, dpkg, python3-yaml,
-                     python3-simplejson, python3-click]
+    stage-packages: [coreutils, bsdmainutils, systemd, dpkg]
     python-packages:
-      - cryptography==3.4.8
       - wheel
       - pip
       - setuptools
```

Reply:
done